dev/PushNotifications/PushNotifications.idl

```diff
@@ -58,11 +58,6 @@ namespace Microsoft.Windows.PushNotifications
     [feature(Feature_PushNotifications)]
     runtimeclass PushNotificationCreateChannelResult
     {
-        PushNotificationCreateChannelResult(
```

Comment:
This should have never been exposed to developers as a projection. The ctor is internal.